test/baseline/User.ts/UserArtist.test.js

```diff
@@ -36,7 +36,25 @@ describe('User.ts/user artist endpoint test', () => {
     expect(attributes.pages).to.eql(0)
     expect(attributes.status).to.eql('ok')
   })
-  it('should get user artists by artist id', async () => {
+
+  //  FIXME: 20221011 mb. Skip.
+  //  Returns error:
+  //    {"status":404,"message":"User is associated to UserGroup using an alias. You've included an alias (user), but it does not match the alias(es) defined in your association (User).","data":null}
+  //  Not clear if:
+  //    - The use case for this endpoint is correct
+  //    - The code for this endpoint is correct
+  //    - The data for this endpoint is correct
+  //
+  //    Can't find an instance of message 'User is associated to UserGroup using an alias. You've included an alias ...' anywhere in the codebase, so
```

Comment:
This is a sequelize error. It means that the way the association [is set up for the user model](https://github.com/resonatecoop/api/blob/main/src/db/models/resonate/user_group.js#L59) does not match [how it's referenced in the endpoint](https://github.com/resonatecoop/api/blob/main/src/controllers/user/artists/routes/%7Bid%7D/index.js#L172). 

Specifically it's complaining about the fact that the query in the route uses `as: 'user'` whereas the model definition doesn't give an `as` alias.